react/features/base/devices/actions.web.ts

```diff
@@ -72,71 +72,62 @@ export function addPendingDeviceRequest(request: Object) {
  */
 export function configureInitialDevices() {
     return (dispatch: IStore['dispatch'], getState: IStore['getState']) => {
```

Comment:
I would add a comment inside the function and also here https://github.com/jitsi/jitsi-meet/blob/master/react/features/conference/actions.web.ts#L56 that we expect getAvailableDevices() to be called before hand. Otherwise I can very well forget about this and remove it by mistake :) 

Otherwise kudos for finding this and changing it. I myself was doing a similar cleanup a year ago probably and there was a lot of `getAvailableDevices` calls happening at the beginning of the call (4-5 IIRC).